### PR TITLE
Docker compose implementation

### PR DESCRIPTION
**Pull request by msperski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,13 @@ docker run -p 8080:8080 --rm -it retroboard:latest
...
 ```
+### Docker-compose
+in root:
+```bash
+mvnw clean package -DskipTests
+cp target/retroboard-0.0.1-SNAPSHOT.jar src/main/docker
+```
+in src/main/docker:
+```bash
+docker-compose run -p 8080:8080 app
```
Comment (golldfish) [inline README.md:23]: If we decide to move docker-compose and Dockerfile to main project directory then docker-compose up --build should also work.

File: src/main/resources/db/migration/V1_0__create_test_schema.sql [hunk 2]
```diff
@@ -1,6 +1,6 @@
-CREATE TABLE IF NOT EXISTS Test(
-    id INTEGER NOT NULL  PRIMARY KEY AUTO_INCREMENT,
+CREATE TABLE IF NOT EXISTS test (
+    id INTEGER NOT NULL  PRIMARY KEY,
     name varchar(20),
     email varchar (50)
 );
```
Comment (MaciejJ0) [inline src/main/resources/db/migration/V1_0__create_test_schema.sql:5]: This table can probably be deleted now

File: src/main/docker/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+services:
+  app:
+    image: 'docker-spring-boot-postgres:latest'
+    build:
+      context: .
+    container_name: app
+    depends_on:
+      - db
+    environment:
```
[the remark this answers is not in the document]
Comment (msperski) [reply src/main/docker/docker-compose.yml:11]: It hits docker-compose variable substitution so I use double $$ but it produces attached error. Im not sure how to solve this. 
image

[the remark this answers is not in the document]
Comment (golldfish) [reply src/main/docker/docker-compose.yml]: I think I was wrong with this one. Sorry for bothered you with that.

**Comment by patrykzawadzkiintive** [comment]
Please do the new structure for local startup and for build & test (used by github actions)

File: src/main/java/com/intive/patronage22/szczecin/retroboard/model/User.java [hunk 4]
```diff
@@ -12,7 +12,7 @@
 @Getter
 @Setter
 @Entity
-@Table(name = "User")
+@Table(name = "users")
```
Comment (daniel-kmiecik) [inline src/main/java/com/intive/patronage22/szczecin/retroboard/model/User.java:15]: Why have you changed all the tables names to lower case?

Comment (msperski) [reply src/main/java/com/intive/patronage22/szczecin/retroboard/model/User.java:15]: Upper letters causes some error in postgres.

**Comment by patrykzawadzkiintive** [comment]
I'm closing this PR. All necessary changes will be committed under new PR

**Pull request closed without merging**